<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gpb2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>681</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>功能区:</string>
    </property>
    <widget class="QLineEdit" name="txtoutput">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>41</y>
       <width>541</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbloutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>42</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>发送:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>651</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,10,1,5,10,10,10,10,10">
      <item>
       <widget class="QLabel" name="lblCOM">
        <property name="text">
         <string>串口号:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbbCOM"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnRefresh">
        <property name="toolTip">
         <string>刷新串口号</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/images/refresh.png</normaloff>:/images/images/refresh.png</iconset>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblrate">
        <property name="text">
         <string>波特率:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbbrate"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpen">
        <property name="text">
         <string>打开串口(&amp;O)</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/images/opencom.ico</normaloff>:/images/images/opencom.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCOM">
        <property name="toolTip">
         <string>发送数据到串口，数据格式为 (JL)(ID16)(DataLen16)[Data](CRC16). 快捷键Alt+W、F5</string>
        </property>
        <property name="text">
         <string>发送数据(&amp;W)</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/images/com.ico</normaloff>:/images/images/com.ico</iconset>
        </property>
        <property name="shortcut">
         <string>F5</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>保存为(&amp;S)</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/images/save.ico</normaloff>:/images/images/save.ico</iconset>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRead">
        <property name="text">
         <string>读取BIN(&amp;B)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="lblinput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>63</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>接收:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtinput">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>63</y>
       <width>541</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gpb1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>配置区:</string>
    </property>
    <widget class="QTreeView" name="treeView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>301</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>471</width>
       <height>321</height>
      </rect>
     </property>
     <property name="title">
      <string>结构体属性:</string>
     </property>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>391</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout"/>
     </widget>
     <zorder>formLayoutWidget</zorder>
     <zorder>gpb1</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
